dracut/30ignition/ignition-setup.sh

```diff
@@ -1,22 +1,29 @@
 #!/bin/bash
+set -eu
 
-set -e
+copy_file() {
```

Comment:
Minor: I'd call this, `copy_file_if_exists()` or something to make it clearer the difference from vanilla `cp`.

Reply:
+1